.gitignore

```diff
@@ -0,0 +1,3 @@
+packaging/.*
+packaging/builds/*
+packaging/targets/*/debian/changelog
```

Comment:
Don't worry, I'l make this change as it's small and I need to upload to the OpenHIE PPA anyways.